content/applications/finance/payment_acquirers/adyen.rst

```diff
@@ -76,10 +83,11 @@ URLs
 ~~~~
 
 To retrieve the URLs, log into your Adyen account, go to :menuselection:`Developers --> API URLs`.
-Pick one of the URLs listed next to **Recurring** as your **Recurring API URL** and one of the URLs
-listed next to **Checkout API** as your **Checkout API URL**.
+Select one of the URLs listed next to :guilabel:`Classic Recurring API` as your
+:guilabel:`Recurring API URL` and one listed next to :guilabel:`Checkout API` as your
+:guilabel:`Checkout API URL`.
 
-.. image:: adyen/adyen_api_urls.png
+.. image:: adyen/adyen-api-urls.png
```

Comment:
The fact that the API names and urls are not aligned in this modified screenshot is triggering me, but not a problem I guess? :laughing:

Reply:
Fixed ;)